src/track/trackmetadatataglib.cpp

```diff
@@ -225,8 +225,15 @@ bool parseBpm(TrackMetadata* pTrackMetadata, QString sBpm) {
     DEBUG_ASSERT(pTrackMetadata);
 
     bool isBpmValid = false;
-    const double bpmValue = Bpm::valueFromString(sBpm, &isBpmValid);
+    double bpmValue = Bpm::valueFromString(sBpm, &isBpmValid);
     if (isBpmValid) {
+        //Some software uses (or used) to write decimated values without comma,
+        //so the number reads as 1352 or 14525 when it is 135.2 or 145.25
+        if (bpmValue > 5000.0) {
```

Comment:
Btw, all bpm values will be rounded to integers when writing ID3v2 tags:

https://github.com/mixxxdj/mixxx/pull/728

That's the standard and we have to follow it. Otherwise you need to keep track internally of the determined bpm scale factor for each track.